view_user_page.php

```diff
@@ -151,5 +153,22 @@
 </div>
 </div>
 </div>
+
+<?php
+if( $t_timeline_view_threshold_access ) {
+	# Build a filter to show all bugs in current projects
+	$g_timeline_filter = array();
+	$g_timeline_filter[FILTER_PROPERTY_HIDE_STATUS] = array( META_FILTER_NONE );
+	$g_timeline_filter = filter_ensure_valid_filter( $g_timeline_filter );
+	$g_timeline_user = $f_user_id;
+	# Override current project, to let timeline show events for all-projects
+	$g_project_override = ALL_PROJECTS;
```

Comment:
Since the timeline on the My View Page based on selected project, I wonder if we should do the same here for consistency.